astropy/utils/iers/iers.py

```diff
@@ -983,8 +983,8 @@ def auto_open(cls, files=None):
         that expires more than 180 - `~astropy.utils.iers.Conf.auto_max_age`
         after the present.
         """
-        good_enough = cls._today() + TimeDelta(180-_none_to_float(conf.auto_max_age),
-                                               format='jd')
+        offset = 180 - (30 if conf.auto_max_age is None else conf.auto_max_age)
```

Comment:
As noted in a previous comment, this has the affect of reverting `auto_max_age` to its default value when it is `None`.  I don't think this was the intended functionality.

Reply:
For future reference: This point has been clarified in subsequent discussions in this PR.